### PR TITLE
Upgrade to Flutter v3.27

### PR DESCRIPTION
**Pull request by Amir-P**

### CODE REVIEW

File: packages/fleather/lib/util.dart [hunk 1]
```diff
@@ -53,3 +53,12 @@ bool isDataOnlyNewLines(Object data) {
   if (data is! String || data.isEmpty) return false;
   return RegExp('^(\n)+\$').hasMatch(data);
 }
+
+int _floatToInt8(double x) => (x * 255.0).round() & 0xff;
```
Comment (amantoux) [inline packages/fleather/lib/util.dart:57]: Don't we want to wrap these function in a `value32Bits` method in an extension on `Color`?

**Review by amantoux — CHANGES_REQUESTED** [review]
Thank Amir! I have a tiny comment PTAL

File: packages/fleather/lib/util.dart (hunk 1 above)
Comment (Amir-P) [reply packages/fleather/lib/util.dart:57]: That makes sense and reads better indeed! @amantoux

**Review by amantoux — APPROVED** [review]
LGTM!
Thanks!

**Pull request merged**